app/src/main/java/org/p2p/wallet/infrastructure/network/provider/TokenKeyProvider.kt

```diff
@@ -43,7 +43,7 @@ class TokenKeyProvider(
         }
     }
 
-    var secretKey: ByteArray = getSecretKeyFromStorage()
+    var keypair: ByteArray = getSecretKeyFromStorage()
```

Comment:
getSecretKeyFromStorage  -> getKeyPairFromStorage

why keypair anyway?

Reply:
We have a big problem which I've figured out.
Will talk in chat